src/compiler/transformers/module/module.ts

```diff
@@ -1206,7 +1207,31 @@ namespace ts {
                         variables = append(variables, variable);
                     }
                     else if (variable.initializer) {
-                        expressions = append(expressions, transformInitializedVariable(variable as InitializedVariableDeclaration));
+                        if (!isBindingPattern(variable.name) && [SyntaxKind.ArrowFunction, SyntaxKind.FunctionExpression].includes(variable.initializer.kind)) {

```

Comment:
`Array.prototype.includes` isn't guaranteed safe to use as the language service can still run in ES5 engines that don't include this method. It would be better to just use `isArrowFunction(variable.initializer) || isFunctionExpression(variable.initializer)` or compare the kinds directly to avoid allocating and throwing away a 2-element array object for a simple comparison.

Reply:
We target ES5, but we unintentionally get ES2015+ features showing up as available due to the fact that `@types/node` is a devDependency.

Reply:
Sounds good, I'll update it to use isArrowFunction/isFunctionExpression.